webserver/__init__.py

```diff
@@ -19,6 +25,9 @@ def create_app():
     import config
     app.config.from_object(config)
 
+    # Run the scheduled jobs
+    scheduledJobs = ScheduledJobs(app.config)
```

Comment:
do you mean "We can start scheduledJobs it a bit later" or "Their must be a better way to do it" ?